src/js/components/navigation.js

```diff
@@ -0,0 +1,66 @@
+import {newContainer} from './helpers'
+
+const renderLinks = (links, type) => {
+  const linkNodes = links.map(link => {
+    const menuLink = newContainer('a',[],['nav__overlay-link']);
+    menuLink.href = link.url;
+
+    if (type === 'image'){
+      const icon = newContainer('img')
+      icon.src = link.image.src;
+      icon.alt = link.image.alt;
+      menuLink.appendChild(icon)
+    } else {
+      menuLink.innerText = link.text;
+    }
+    const menuItem = newContainer('li',[menuLink]);
+    return menuItem;
+  })
+  return linkNodes;
+}
+const navigationComponent = (data) => {
+  //Header logo and expand menu.
+  const imageLogo = newContainer('img');
+  imageLogo.src = data.image.src;
+  imageLogo.alt = data.image.alt;
+
+  const logoLink = newContainer('a',[imageLogo],['nav__image']) 
+  logoLink.href = data.logoUrl;
+
+  const imageMenu = newContainer('img')
+  imageMenu.src = 'menu.png';
+  imageMenu.alt = 'menu icon';
```

Comment:
load this information from the data

Reply:
it is coming from the assets not need to be added in the data